tests/conftest.py

```diff
@@ -44,7 +44,7 @@ def _parse_command(self, cmd):
             args = shlex.split(cmd)
         return args
 
-    def _checked_call(self, cmd, expect_success=True, expect_error_output=False):
+    def unchecked_call(self, cmd, log_output=True):
```

Comment:
I'm not too happy with the new name. While it makes sense in the context of the diff, it doesn't on its own, because it's not even clear anymore what "checking" is supposed to mean when just looking at the function.

That said, I thought about it and couldn't come up with a much better suggestion. If it was still internal, I'd call it `_call`, but it's not. So maybe just add a docstring along the lines of "Performs a herbstclient call without checking for its success"?

Reply:
I've added a doc-string here and to the `call()`, which checks there exit code.